.travis.yml

```diff
@@ -94,7 +94,7 @@ install:
       sudo apt-key add -
 
   # Update the package list and install the Cloud SDK
-  - sudo apt-get update
+  - sudo apt-get -qq update
```

Comment:
will this print something in case of error? `qq` suppresses everything right?

Reply:
According to the help: `-qq No output except for errors`. So it should be safe, otherwise we could limit to only -q which will be more verbose of course.

Reply:
That's optimal then 👍